attentive-android-sdk/src/main/java/com/attentive/androidsdk/AttentiveApi.java

```diff
@@ -34,8 +42,54 @@ public AttentiveApi(OkHttpClient httpClient, ObjectMapper objectMapper) {
         this.objectMapper = objectMapper;
     }
 
+    // TODO refactor to use the 'sendEvent' method
     public void sendUserIdentifiersCollectedEvent(String domain, UserIdentifiers userIdentifiers, AttentiveApiCallback callback) {
         // first get the geo-adjusted domain, and then call the events endpoint
+        getGeoAdjustedDomainAsync(domain, new GetGeoAdjustedDomainCallback() {
+            @Override
+            public void onFailure(String reason) {
+                callback.onFailure(reason);
+            }
+
+            @Override
+            public void onSuccess(String geoAdjustedDomain) {
+                internalSendUserIdentifiersCollectedEventAsync(geoAdjustedDomain, userIdentifiers, callback);
+            }
+        });
+    }
+
+    public void sendEvent(Event event, UserIdentifiers userIdentifiers, String domain) {
+        sendEvent(event, userIdentifiers, domain, null);
+    }
+
+    public void sendEvent(Event event, UserIdentifiers userIdentifiers, String domain, @Nullable AttentiveApiCallback callback) {
+        getGeoAdjustedDomainAsync(domain, new GetGeoAdjustedDomainCallback() {
```

Comment:
are were moving this call to sdk instantiation as part of the purchase events stuff?

Reply:
oh nvm i see your comment below

Reply:
yeah since Purchases are low volume it's not necessary right now. Will be added with the other events.